MITADS/opensubtitles_exporter.py

```diff
@@ -0,0 +1,367 @@
+
+  ##space trim - it must be reapplied again
+  [ re.compile('\s+'), u' ' ]
+]
+
+
+mapping_normalization_after_decode = [
+
+  ## Convert old fashion accented letter to the real accented letter
+  [ u'E\' ', u'è ' ],
+  [ re.compile('a\'(\s|$|,|\.|\?)'), r'à\1' ], 
+  [ re.compile('e\'(\s|$|,|\.|\?)'), r'è\1' ],
+  [ re.compile('i\'(\s|$|,|\.|\?)'), r'ì\1' ],
+  [ re.compile('o\'(\s|$|,|\.|\?)'), r'ò\1' ],
+  [ re.compile('u\'(\s|$|,|\.|\?)'), r'ù\1' ],     
+
+]
+
+
+def maybe_normalize(value, mapping=mapping_normalization):
+  for norm in mapping:
+    if(value==''):
+      break
+
+    if type(norm[0]) == str:
+      value = value.replace(norm[0], norm[1])
+    elif isinstance(norm[0], Pattern):
+      try:
+        value = norm[0].sub(norm[1], value)
+      except Exception as e:
+        raise(e)
+    else:
+      print('UNEXPECTED', type(norm[0]), norm[0])
+
+  return value
+
+def line_not_relevant(text):
+  
+  if(len(text)<2):
+    return True
+
+  
+  ## If the sentence include those symbols we ignore it (is part of the opening subtitles)  
+  if('®' in text
+     or '©' in text
+     or '±' in text):
+    return True
+
+  ##sentence with one word that contain those symbols, we ignore it
+  if(not ' ' in text and 
+    (   '_' in text
+     or '@' in text
+     or '}' in text     
+     or '+' in text
+     or '{' in text)):
+     return True
+
+  ##add other cases
+
+  return False
+ 
+
+def normalize_text(text):
+
+  ## If the whole sentence is uppercase we convert to a lowercase version
+  if(text.isupper()):
+    text = text.lower()
+
+  #normalization first step (replace and sub regex)
+  text = maybe_normalize(text,mapping_normalization)
+
+  ###### UNICODE/NO-ASCII CHAR
+  ##Opensubtiles Dataset contains no-ASCII char
+  ## instead of process single unicode case we use unidecode package to delegate all unicode char processing
+  # to keep all vowels properly accented, and at the same time eliminate the other unicode characters, you need to use a substitution with place holders
+  ##abstent unicode placeholder
+  text =  text.replace('à', '<PH_A>')
+  text =  text.replace('è', '<PH_E>')
+  text =  text.replace('ì', '<PH_I>')
+  text =  text.replace('ò', '<PH_O>')
+  text =  text.replace('ù', '<PH_U>')
+
+  text = unidecode(text)## # Transliterate an Unicode object into an ASCII string
+  ###
+  text =  text.replace('<PH_A>','à')
+  text =  text.replace('<PH_E>','è')
+  text =  text.replace('<PH_I>','ì')
+  text =  text.replace('<PH_O>','ò')
+  text =  text.replace('<PH_U>','ù')
+  #############################################################
+  
+  ##normalization second step:  that produces output with unicode characters
+  text = maybe_normalize(text,mapping_normalization_after_decode)
+
+  text = text.strip()
+
+  if(len(text)==0):
+    return ''
+  #################################
+  ##other normalization step
+
+  ##remove dot and apex only if at the beginning of sentence
+  if(text[0]=='.' or text[0]=='\''): 
```

Comment:
```suggestion
  if(text[0]=='.' or text[0]==':' or text[0]=='\''):
```

To handle sentences that star with that symbol